airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1847,6 +1847,16 @@ definitions:
       type:
         type: string
         enum: [RemoveFields]
+      condition:
+        description: The predicate to filter a property by a property value. Property will be removed if evaluated to True.',
```

Comment:
Probably worth mentioning the fields will always be removed if the condition field is not set

Reply:
added